core/lib/config/src/configs/observability.rs

```diff
@@ -1,6 +1,8 @@
+use serde::Deserialize;
+
 /// Configuration for the essential observability stack, like
 /// logging and sentry integration.
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Deserialize)]
```

Comment:
I'm using `ObservabilityConfig` in `ProverAutoscalerConfig`, which I deserialize with serde directly now, so it require for all the structs to implement `Deserialize`. Before it wasn't needed because we deserialize generated from protobuf struct and then converted into config structs.